src/App.js

```diff
@@ -1,25 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
+import SignButton from './components/Buttons/BasicSigns';
+import NumberButton from './components/Buttons/NumberButton';
+import CalcInput from './components/CalcInput';
+import { numbers, signs } from './static/button';
 
 function App() {
   return (
```

Comment:
- [ ] <b>REQUIRED:</b> The `App.js` component should be class-based. Kindly fix that.